src/fty_sensor_gpio_assets.cc

```diff
@@ -578,7 +602,7 @@ fty_sensor_gpio_assets_test (bool verbose)
     zstr_sendx (assets, "CONNECT", endpoint, NULL);
     zstr_sendx (assets, "CONSUMER", FTY_PROTO_STREAM_ASSETS, ".*", NULL);
     // Use source-provided templates
-    zstr_sendx (assets, "TEMPLATE_DIR", "./data/", NULL);
+    zstr_sendx (assets, "TEMPLATE_DIR", test_data_dir, NULL);
```

Comment:
@jimklimov not sure this will work! Conversely to the other actors tests which use SELFTEST_DIR_RW, this assets and its tests need to access the template dir for verification, not to generate something into the data dir...